indico/legacy/webinterface/tpls/RoomBookingNewBookingCollaborationWidget.tpl

```diff
@@ -18,23 +23,29 @@
     </div>
     <ul id="videoconferenceOptions" class="weak-hidden">
         % for subfield in [form.needs_vc_assistance] + eq_list:
-            <li class="js-vc-row">
-                ${ subfield() }
+            % if subfield.short_name == 'needs_vc_assistance' and start_dt.time() < work_start or start_dt.time() > work_end:
+                <li class="js-vc-row unavailable" title="${ _('Assistance is not available because start time of booking is not within working hours.') }">
+                    ${ subfield(disabled=True) }
+            % else:
+                <li class="js-vc-row">
+                    ${ subfield() }
+            % endif
                 ${ subfield.label(style='font-weight: normal;') }
```

Comment:
For the sake of readability I'd duplicate this and the next line in both the if and else blocks.